src/configSchema.ts

```diff
@@ -100,7 +100,7 @@ export const VALIDATION_SCHEMA = z
 		delay: z.number().nonnegative({
 			message: ZodErrorMessages.delay,
 		}),
-		torznab: z.array(z.string().url()),
+		torznab: z.array(z.string().url()).nullish(),
```

Comment:
how about just an empty array?

Reply:
I figured that would more likely be an error/misconfiguration compared to deliberately setting undefined or null. Mainly because the default is an empty array.

Reply:
Maybe a note in the config description about this would be appropriate?

Reply:
we still accept an empty array though, so *also* allowing null doesn't really prevent that misconfiguration from happening. imo the user will still see the warning if it is an accident, and then in the code we don't have to worry about torznab ever being null.